tensorrt_llm/_torch/cutedsl_matmul_heuristics.py

```
@@ -0,0 +1,277 @@
+    interface = _get_interface(precision)
+    if interface is None:
+        return []
+    layout = _get_layout(NVFP4_LAYOUT)
+    if layout is None:
+        return []
+    try:
+        # hw=None targets the current GPU.
+        try:
+            interface.loadInternalDiscoverySet(layout, None)
+        except Exception:  # noqa: BLE001 - discovery data is optional
+            pass
```

Ruff (0.15.20)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)